tools/benchmark_tool/requirements.txt

```diff
@@ -1,2 +1,2 @@
--c ../constraints.txt
 numpy>=1.16.6,<2.1.0
+build<1.3
```

Comment:
I suppose `requirements` are supposed to be `deployment`, but `build` ones

Reply:
Removed `build` from requirements, should no longer be an issue.